neurovault/apps/statmaps/views.py

```diff
@@ -670,7 +669,7 @@ def atlas_query_region(request):
         else:
             synonymsDict = {}
             with open(os.path.join(neurovault_root, 'neurovault/apps/statmaps/NIFgraph.pkl'),'rb') as input:
```

Comment:
I think so, I would have gotten a lot of errors if that weren't the case, and here is a quick test:

      import pickle
      from sklearn.externals import joblib
      data = [1,2,3]
      pickle.dump(data,open("/tmp/pickle.pkl","wb"))
      joblib.load("/tmp/pickle.pkl")
      [1, 2, 3]